cmd/derper/derper.go

```diff
@@ -226,7 +230,7 @@ func main() {
 		}
 		go func() {
 			port80srv := &http.Server{
-				Addr:         ":80", // the default, but to be explicit
+				Addr:         listenHost + ":80", // the default, but to be explicit
```

Comment:
use net.JoinHostPort, not +, or you'll construct invalid IPv6 listen addrs.

Reply:
Thank you for your advice.